hr_timesheet_task_required/tests/test_hr_timesheet_task_required.py

```diff
@@ -32,31 +32,41 @@ def setUpClass(cls):
             'project_id': cls.project_2.id,
         })
 
+        cls.employee = cls.env['hr.employee'].create({
+            'name': "Test User",
+            'user_id': cls.user.id,
+            'company_id': cls.user.company_id.id,
+        })
+
     def test_timesheet_line_task_required(self):
         with self.assertRaises(ValidationError):
             self.AnalyticLine.create({
                 'name': 'test',
                 'project_id': self.project_1.id,
                 'unit_amount': 10,
+                'employee_id': self.employee.id,
```

Comment:
Something in hr_timesheet_sheet triggered this?

Reply:
No. Branch 12.0 seems broken. See PR https://github.com/OCA/hr-timesheet/pull/212, travis fails as well.

The cause is that module `hr_timesheet_employee_required` is installed before module `hr_timesheet_task_required`.

Reply:
I'll create a separate PR if tests become green.

Reply:
Riiight, my bad - hr_timesheet_employee_required has to be off by default

Reply:
To avoid side effects

Reply:
Yet that makes no sense.....

Reply:
Ok, then it could be fixed directly in travis config file. :+1:

Reply:
Opened a dedicated PR https://github.com/OCA/hr-timesheet/pull/213 for the tests of `hr_timesheet_task_required`.